.github/workflows/upgrade-extensions.yml

```diff
@@ -23,6 +23,8 @@ jobs:
     - run: npm install
     - run: npm install --global yarn
     - run: node upgrade-extensions
+      env: 
+        GH_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Could we update the secret name to remove reference to the personal access token?

Reply:
Donezo, renamed it in [`ded9bb4` (#472)](https://github.com/open-vsx/publish-extensions/pull/472/commits/ded9bb4b0b487396e0cdc77bc134807ea9abac38)